_posts/2018-04-03-EDK2-based-firmware-on-MinnowBoard.md

```diff
@@ -0,0 +1,56 @@
+Building EDK2 based firmware for MinnowBoard
+============================================
+
+There are some options to build firmware for MinnowBoard, a Bay Trail based
+SBC (Single Board Computer) from Intel. We prefer usually coreboot as simplest
+and fastest, open source solution, but sometimes we want to have UEFI
```

Comment:
There are some options to build firmware for MinnowBoard, a Bay Trail-based SBC (Single Board Computer) from Intel. We usually prefer coreboot as the simplest and fastest open source solution, but sometimes we want to have UEFI interface.